### ISSUE TITLE
consider how to make share, bookmark, and report an issue buttons more obvious for screen readers

### ISSUE THREAD

**Issue by Smullz622** [psu-libraries/psulib_blacklight#1395]
The share, bookmark, and report an issue button on an individual record page are easy to miss with the screen reader. Typical screen reader use would be to jump to the record title and then tab through, skipping over those buttons entirely. Michelle's suggestion was to move them lower to be below the initial record info. I'm hesitant to do that from a general UI perspective so my instinct is to leave it as is. To be clear, the screen reader can still reach those buttons, you just have to tab through to get to them.

Ruth's feedback:
I agree with you that I hesitate to move it from a UX perspective, especially with everything else going on. It feels like some of the challenge is something we face with every user which is "how much are they going to skim the site and how much are they going to pay attention to what they could actually do." e.g. if/once a screen reader user notices we have Bookmark, they should be able to navigate to it and we'll add the better alt text so they can be sure it applies to the right item. Which feels not unlike pointing out something on a screen that a visual user never knew we had even though they've used it a hundred times because they're always skimming.

**Comment by Smullz622** [comment]
Let's try doing an alt text header around these buttons: 'Actions for [Title]'

**Comment by granta03** [comment]
One possible solution I found is using the `tabindex` keyword to have explicit navigation order. One source I found says to avoid using it though because it can cause issues with the rest of the page and its tabbing order. [https://webaim.org/techniques/keyboard/tabindex](url)

**Comment by granta03** [comment]
@Smullz622 I added the header you requested. However I noticed that when tabbing the bookmark button is skipped. When using ctrl+option+arrow keys it can be highlighted though.
